desktop_app/src/backend/server/plugins/llm/index.ts

```diff
@@ -109,6 +109,10 @@ const llmRoutes: FastifyPluginAsync = async (fastify) => {
           model: await createModelInstance(model, provider),
           messages: convertToModelMessages(messages),
           stopWhen: stepCountIs(vercelSdkConfig.maxToolCalls),
+          experimental_transform: smoothStream({
+            delayInMs: 20, // optional: defaults to 10ms
+            chunking: 'line', // optional: defaults to 'word'
+          }),
```

Comment:
I tried with and without this enabled -- my experience _with_ this change was I didn't really see the streaming of tokens into the "Assistant" chat message, it was all rendered at once. I also tried changing `chunking` back to `word` and using the 20ms "delay", still felt a bit off.

Did you have much luck with this change? Maybe we play around with this in a separate PR?